src/screens/Classrooms.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react'
+import { useForm } from 'react-hook-form'
+import 'firebase/firestore'
+
+import { useFirestore, useFirestoreCollectionData, useUser } from 'reactfire'
+
+import { FormErrorMessage, FormLabel, FormControl, Input, Button } from '@chakra-ui/react'
+
+import Classroom from 'components/Classroom'
+
+const Classrooms: React.FC = () => {
+  const { handleSubmit, errors, register, formState } = useForm<FormData>()
+
+  const user = useUser()
+
+  type FormData = {
+    classroomName: string
+  }
+
+  //Steve is the coolest
+  interface IClassroom {
+    docId: string
+    classroomName: string
+    students: []
+  }
+
+  const classroomRef = useFirestore().collection('classrooms')
+  const classroomDocuments = useFirestoreCollectionData<IClassroom>(classroomRef, { idField: 'docId' })
+  console.log({ classroomDocuments, user })
+
+  async function onSubmit(values: FormData) {
```

Comment:
syntax here is overkill since you're not doing anything after your await.  you may as well use the promise API and simply not invoke `then`

e.g.
`classroomRef.add({...etc}).catch(//*handle your error *//)`

Reply:
That makes sense.  Will do

Reply:
@DoctorBurglar  done yet?